docs/_docs/troubleshooting.md

```diff
@@ -97,7 +97,7 @@ Xcode.app can interfere with the command line tools downloaded above. If
 you run into this issue, upgrade Xcode and install the upgraded Command
 Line Tools.
 
-### Running Jekyll as Non-Superuser (no sudo!)
+### Running Jekyll as Non-Superuser (no sudo!) <a id="no-sudo"></a>
```

Comment:
We use Kramdown's block attributes in our docs to add anchors: https://kramdown.gettalong.org/quickref.html#block-attributes